public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -84,7 +82,7 @@ include ../../../../_includes/_util-fns
 :marked
   Now we create a car by passing the engine and tires to the constructor.
 
-+makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', 'Simple Car')(format=".")
++makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', '')(format=".")
```

Comment:
In all the pages I looked at before, we were using this title to specify the filename and whether it's an excerpt or a specific version.

It seemed weird to have a different convention here.